src/main/java/ch/wisv/areafiftylan/controller/AuthenticationController.java

```diff
@@ -61,6 +72,19 @@ public ModelAndView getLoginPage() {
         return createResponseEntity(HttpStatus.OK, "Here's your token!");
     }
 
+    @RequestMapping(value = "/token", method = RequestMethod.POST)
+    public ResponseEntity<?> createAuthenticationToken(@RequestBody UserDTO userDTO) {
+        String authToken = authenticationService.createNewAuthToken(userDTO.getUsername(), userDTO.getPassword());
+
+        return createResponseEntity(HttpStatus.OK, "Token successfully created", authToken);
+    }
+
+    @PreAuthorize("isAuthenticated()")
+    @RequestMapping(value = "/token/verify", method = RequestMethod.POST)
+    public ResponseEntity<?> verifyToken() {
+        return createResponseEntity(HttpStatus.OK, "Token is valid!");
```

Comment:
If this endpoint is a POST, it seems an action happens. That a token actually gets verified. From reading the response it seems this is just getting the validity of a token. No action happens upon reaching this endpoint. I think this should be a GET endpoint.